### ISSUE TITLE
The requested module '{p5 module}' does not provide an export named 'default'

### ISSUE THREAD

**Issue by drac-dot-gay** [tonyketcham/p5-svelte#83]
**Describe the bug**

**To Reproduce**
1. Create a new sveltekit project with the latest version of p5-svelte and p5
2. Set up the p5 component as listed in the sapper instructions
3. observe how nothing appears in the browser and the devtools console has an error about module exports

**Expected behavior**
The p5 canvas running properly

**Screenshots**
![image](https://user-images.githubusercontent.com/8054329/131238826-c77c1b02-4b59-4ff3-95e5-431fddc81451.png)

**Desktop**
 - OS: Windows 11
 - Browser Chrome
 - Version 92.0.4515.159

**Comment by muhsin61** [comment]
I didn't use npm and I added this as a component and it works. 
- via [/src/P5.svelte](https://github.com/tonyketcham/p5-svelte/blob/9fc8306b8d3369655e06eb0cae731dc610f56897/src/P5.svelte)

`lib> p5-svelte.svelte`
```js
<script>
    import { onMount, createEventDispatcher } from 'svelte';
    // API properties
    export let project = undefined;
    export let target = undefined;
    export let sketch = undefined;
    // Event generation
    const event = createEventDispatcher();
    const dispatch = {
      ref() {
        event('ref', target);
      },
      instance() {
        event('instance', project);
      },
      p5() {
        event('');
      },
    };
    /**
     * Creates a reference for the p5 instance to render within
     * @param {HTMLElement} node
     */
    function ref(node) {
      target = node;
      return {
        destroy() {
          target = undefined;
        },
      };
    }
    function augmentClasses(instance, classes) {
      classes.forEach(([key, value]) => (instance[key] = value));
      return instance;
    }
    /**
     * When the client loads, create the p5 instance
     */
    onMount(async () => {
      const library = await import('p5');
      const { default: p5 } = library;
      const entries = Object.entries(library);
      const nativeClasses = entries.filter(
        ([key, value]) =>
          value instanceof Function && key[0] !== '_' && key !== 'default'
      );
      project = new p5((instance) => {
        instance = augmentClasses(instance, nativeClasses);
        console.log(instance);
        return sketch(instance);
      }, target);
      // Initial event dispatching
      dispatch.ref();
      dispatch.instance();
    });
  </script>
  
  <div use:ref />
  
  <style>
    div {
      display: inline;
      margin: 0;
    }
  </style>
```
  
And  
```js
onMount(async () => {
  const module = await import("../lib/p5-svelte.svelte");
  P5 = module.default;
});
```

**Comment by tonyketcham** [comment]
@muhsin61 yup! This is a preview of v3.0 which isn't yet released via npm but is a release candidate on the main branch. It should work for everything in the Svelte ecosystem, but I still need to investigate how tree-shaking may be impacted by the mutation of the p5 library where I've merged all native classes in:
```js
project = new p5((instance) => {
  instance = augmentClasses(instance, nativeClasses);
  console.log(instance);
  return sketch(instance);
}, target);
```

That allows for p5 classes like `Vector` to work without special imports, however this solution may bloat the build for end-users.

**Comment by tonyketcham** [comment]
Resolved in [`v3.0`](https://github.com/tonyketcham/p5-svelte/releases/tag/v3.0.0) 🎉
